app/controllers/api/base_controller.rb

```diff
@@ -1,3 +1,26 @@
 class Api::BaseController < ApplicationController
   layout nil
-end
+
+  def authenticate_user
+    email = params[:email]
```

Comment:
Consider adding input validation for email and password to prevent potential security issues such as SQL injection or invalid data being processed.